alexa-sdk/alexa-sdk.d.ts

```diff
@@ -0,0 +1,132 @@
+// Type definitions for Alexa SDK for Node.js v1.0.3
+// Project: https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs
+// Definitions by: Pete Beegle <https://github.com/petebeegle>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'alexa-sdk' {
+    export function handler(event: RequestBody, context: Context, callback?: Function): AlexaObject;
+    export function CreateStateHandler(state: string, obj: any): any;
+    export var StateString: string;
+
+    interface AlexaObject {
+        _event: any;
+        _context: any;
+        _callback: any;
+        state: any;
+        appId: any;
+        response: any;
+        dynamoDBTableName: any;
+        saveBeforeResponse: boolean;
+        registerHandlers: (...handlers: Handlers[]) => any;
+        execute: () => void;
+    }
+
+    interface Handlers {
+        [intent: string]: () => void;
```

Comment:
note for 2.0: the type should be `(this: Handler) => void` once this-parameters are available.